tests/layer_tests/pytorch_tests/test_divmod.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy.testing as npt
+import openvino as ov
+import pytest
+import sys
+import torch
+
+
+# do not test via PytorchLayerTest since PytorchLayerTest triggers own TorchScript tracing
+# this test validates TorchScript patched inside ov.convert_model
+class TestBuiltinDivmod():
+    class builtin_divmod(torch.nn.Module):
+        # test divmod() on assert path that is omitted in ov::Model
+        def forward(self, x):
+            divisor, rem = divmod(x.shape[-1], 7)
+            assert rem == 0, x.shape
+            assert divisor > 0 and (divisor & (divisor - 1)) == 0, (divisor, x.shape)
+            new_shape = tuple(x.shape[:-1]) + (-1, 7)
+            return x.reshape(new_shape)
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize("fw_model,inputs", [
+        (builtin_divmod(), torch.randn(2, 3, 28)),
```

Comment:
Do not use random in test arguments, it breaks parallelization

Reply:
Done, great catch!